### ISSUE TITLE
[BUG] subtitle embedded into the video

### ISSUE THREAD

**Issue by demonic-avenger** [anilbeesetti/nextplayer#849]
Okay, so I think I've fully identified the problem. If the video is in any format other than .mp4 then embedded subtitle wouldn't show up.
I've tried it in .mkv and .mov, the embedded subtitle didn't show up, but in .mp4 it worked just fine. I'm giving a link three videos, they are the same clip but in different format.
https://mega.nz/folder/01ERGRDY#jFEnwMQzWRGq30B90M2QVQ

And this is what it's actually looks like
![Screenshot_20240523-122034_Next_Player](https://github.com/anilbeesetti/nextplayer/assets/79894684/cee0b15b-442f-4a8d-b4bb-d2d2f00337d9)

**Comment by anilbeesetti** [comment]
@demonic-avenger Could you please share the video sample in which this problem is observed?

**Comment by Tanmandal** [comment]
https://mega.nz/file/J5IBSBba#Rgge9eU5fbAkHOm65ieDvg9xmVC-BqABYNyxkKv-j6o

Check this video. Embedded subtitles are not shown.
I have also checked with vlc . There it works fine

**Comment by demonic-avenger** [comment]
> @demonic-avenger Could you please share the video sample in which this problem is observed?

Right, so I figured something more out about the issue, if video format is .mkv then the embedded subtitle wouldn't show, but if it's in .mp4 then it would show, I don't know about any other format through, any way I'll update my issue

**Comment by Onxr** [comment]
Same here. If it's mkv the sub doesn't show but it's working if I use VLC.

**Comment by sandenbergmelo** [comment]
Same here.
